UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Helpers/IFileSystemHelper.cs

```diff
@@ -41,5 +41,9 @@ public interface IFileSystemHelper
         bool CreateEmptyFileContent(string filePath);
 
         bool DownloadReadmeFile(string filePath, Stream fileStream);
+
+        Task CreateXmlFromObject<T>(T obj, string filePath, string fileName);
```

Comment:
do we need Task here? can we make it void

Reply:
To achieve unit test added return type as Task.